scripts/base36-uuid.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023-present ScyllaDB
+#
+
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+
+import argparse
+import datetime
+import re
+import string
+import uuid
+
+# If "uuid_sstable_identifiers_enabled" is set, scylla generates sstables using
+# timeuuid as their identifiers. But the string representation does not follow
+# the formal definition defined by rfc4122. Instead, it uses a base36-based
+# presentation which is used by Cassandra. In comparison to the rfc4122
```

Comment:
I'm afraid to admit I don't understand what you mean a "base36-based presentation which is used by Cassandra". Can you give an example of such string? And it's used where? In normal printing of "timeuuid" variables? Or somewhere else?

Reply:
could you please take a look https://github.com/scylladb/scylladb/pull/14374/files#diff-155198c45dedd63344b6602fe0aa451682e64a0185e2a3b8167ecf3a0684ef6fR170 ? it is used in the place of the generation of sstable. like "nb-3fw2_0tj4_46w3k2cpidnirvjy7k-big-Data.db".

Reply:
I suggest that you please give this example ""3fw2_0tj4_46w3k2cpidnirvjy7k in the commit message. My next question was that you claimed it is used "by Cassandra". Where? Cassandra documentation like https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/timeuuid_functions_r.html that print timeuuid print it in a different format, which seems to have base 16, not 36, minus signs, not underscores, and differently split into parts...

Reply:
I just now realized (because you explained it to me in a reply - this explanation was missing in the commit message) that Cassandra recently added a *second* way to print timeuuids, specifically for the sstable names, and it's not the same as the original base16 printing of timeuuids.